vars/abapEnvironmentPipelineStagePrepareSystem.groovy

```diff
@@ -0,0 +1,27 @@
+import groovy.transform.Field
+import com.sap.piper.Utils
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field STAGE_STEP_KEYS = []
```

Comment:
see comment above

> please maintain STAGE_STEP_KEYS, this is used for the documentation.
You can find an example here:
https://github.com/SAP/jenkins-library/blob/master/vars/piperPipelineStageAcceptance.groovy#L11